requirements.txt

```diff
@@ -1,5 +1,6 @@
 pbr>=2.0 # Apache-2.0
 ansible>=2.9.0,<2.11.0,!=2.9.8,!=2.9.12 # GPLv3
+ansible-base@git+https://github.com/stackhpc/ansible@stackhpc/2.10/rocky # GPLv3
```

Comment:
I wonder if we could use an environment marker to apply this only to Rocky?

https://peps.python.org/pep-0508/

Reply:
Interesting idea. I guess that would work if you matched the OS on your control host with the hosts targeted by ansible, but if for instance you tried to use a ubuntu control host to configure a Rocky host, wouldn't you end up install a version of ansible without Rocky support. 

Or were you thinking more an environment variable that would switch the behaviour? I think in the AIO one jobs we share the same kayobe docker image, so we might end up using this version of ansible for all jobs.

Reply:
Yes, I suppose you're right there.

My hesitation was in using a stale version of ansible, but actually it's EOL, and you have the final commit in that branch.